src/components/wizard_steps/InstallationPathSelect.vue

```diff
@@ -27,8 +27,9 @@ export default {
     const installPath = ref('');
 
     onMounted(async () => {
-      const homePath = await homeDir();
-      installPath.value = await path.join(homePath, '.espressif');
+      const path = await invoke("get_installation_path");
```

Comment:
What happens if invoke fails, for example an error is generated ?

Reply:
this is actually polling app state, there is almost no logic which could fail. In case the app state is locked (mutex) the user will get error message in top right corner of the app similar to this:
![image](https://github.com/user-attachments/assets/99ca1710-2e27-4983-bfac-5f645242671f)